test/test_lib/KeyValueStoreStub.sol

```diff
@@ -0,0 +1,225 @@
+pragma solidity ^0.4.23;
+
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import "../../contracts/gateway/Hasher.sol";
+import "../../contracts/gateway/MessageBus.sol";
+
+/**
+ * @title Stub data used across tests.
+ */
+contract KeyValueStoreStub {
+
+    mapping(bytes32 => uint256) public uintStorage;
+    mapping(bytes32 => address) public addressStorage;
+    mapping(bytes32 => bytes32) public bytes32Storage;
+    mapping(bytes32 => bytes) public bytesStorage;
+
+    Hasher hasher = new Hasher();
+    MessageBus.MessageBox messageBox;
+    MessageBus.Message message;
+
+    /* Special Functions */
+
+    /**
+     * @notice contract constructor.
+     *
+     * @dev It sets the stub data based on their data type in respective
+     *      mapping
+     */
+    function KeyValueStoreStub()
+        internal
+    {
+        // Stores uint256
+        setUint("NONCE", 1);
```

Comment:
uint is alias for uint256, but for the consistency with other code and future compatibility, we can rename this to `setUint256` ?